### PR TITLE
tidy up for going live

### PR DESCRIPTION
**Pull request by whoburg**
This is by no means finished; this is just a first stab at making the site presentable and removing entirely blank pages for now.

Ready for review.

### CODE REVIEW

**Comment by whoburg** [comment]
done

File: index.md [hunk 1]
```diff
@@ -3,4 +3,16 @@ layout: default
 title: Home
 ---
 
-Content coming soon.
+Tackling engineering design problems using tools from convex optimization and computer science.
+
+### Recent posts
+<div>
```
Comment (bqpd) [inline index.md]: maybe with `class="posts"` just for consistency, since I just realized that's done elsewhere.

**Comment by bqpd** [comment]
One comment, otherwise ready to merge.

File: index.md (hunk 1 above)
Comment (whoburg) [inline index.md]: done. I'm going to squash all these oneliner commits and force push to this branch -- standby

**Comment by whoburg** [comment]
ok cleaned up the commit hx and addressed comments, @bqpd please merge if happy

**Pull request merged**